PrivateCollections/Ethiopia/MamherDegguAddet/MDA003.xml

```diff
@@ -30,42 +31,260 @@
+                                          <locus from="35r" to="42v"></locus>
+                                      </item>
+                                      <item xml:id="q7" n="6">
+                                          <num value="6">፮</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="43r" to="50v"></locus>
+                                      </item>
+                                      <item xml:id="q8" n="7">
+                                          <num value="7">፯</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="51r" to="58v"></locus>
+                                      </item>
+                                      <item xml:id="q9" n="8">
+                                          <num value="8">፰</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="59r" to="66v"></locus>
+                                      </item>
+                                      <item xml:id="q10" n="9">
+                                          <num value="9">፱</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="67r" to="74v"></locus>
+                                      </item>
+                                      <item xml:id="q11" n="10">
+                                          <num value="10">፲</num>
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="75r" to="83v"></locus>
+                                          <note>The folio 77 was photographed twice as 77 and 78.</note>
+                                      </item>
+                                      <item xml:id="q12" n="11">
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="84r" to="91v"></locus>
+                                      </item>
+                                      <item xml:id="q13" n="12">
+                                          <dim unit="leaf">8</dim>
+                                          <locus from="92r" to="99v"></locus>
+                                      </item>
+                                      <item xml:id="q14" n="13">
+                                          <num value="2">፪</num>
+                                          <dim unit="leaf">6</dim>
+                                          <locus from="100r" to="105v"></locus>
+                                      </item>
+                                      <item xml:id="q15" n="14">
+                                          <dim unit="leaf">7</dim>
+                                          <locus from="106r" to="112v"></locus>
+                                          6, stub after 1
```

Comment:
Is it possible to note it like this? I am always writing it like this: "s.l. 6, stub after 1"

Reply:
yes, both are possible, I think there is no difference